working-together/README.md

```diff
@@ -0,0 +1,23 @@
+Working Together
+==============
+
+A guide for working well together.
+
+
+Pair Programming
+-------
+
+**Before:**
+* Agree Scope (Pairing for two hours? Until the ticket is complete? Just get pass the bug?)
+* Agree Working Environment (Two keyboards? VIM/what other tools?)
+* Agree Pairing Style (Time-based? Ping pong? To the next traffic light?)
+
+**During:**
+* Keep the chat going (Think aloud. Encourage/support)
+* Keep switching (Follow the pairing style)
+* Keep both involved ("Could we do this another way?")
+* Keep breaking (Have a break, have a…)
```

Comment:
Should there be additional text after "have a...."? Possible suggestions: "Walk to the kitchen together for a snack/tea/coffee. Take a moment to stand and stretch. Go for a short walk."

Reply:
haha, I *thought* this might be KitKat related but I've never seen that particular commercial.